admin_honeypot/listeners.py

```diff
@@ -1,7 +1,7 @@
 from admin_honeypot.signals import honeypot
 from django.conf import settings
 from django.core.mail import mail_admins
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```

Comment:
This will work:

```
try:
    from django.urls import reverse
except ImportError:  # Django<2.0
    from django.core.urlresolvers import reverse
```